### DISCUSSION TITLE
Should I turn off my codespace when I am done using it ?

### DISCUSSION THREAD

**Discussion by ajnart** [community/community#17019] [Codespaces] [answered]
It's all in the title ! Should I stop my codespace when it's not in use or can I keep it running ? Are the costs lower if it's not in use ?
The automatic port forwarding is awesome btw !

**Comment by batmanscode** [comment]
FYI it automatically shuts down after 30 minutes (can be changed) of inactivity

**Comment by treeder** [comment, marked as answer]
If you want to save money on the 30 minutes until auto-shutdown, then yes you should. If you don't care about the extra 30 minutes, then no need to shut it down manually.

**Reply by ajnart** [reply]
> If you want to save money on the 30 minutes until auto-shutdown, then yes you should. If you don't care about the extra 30 minutes, then no need to shut it down manually.

Thank you for the reply. I don't think I have any payment methods linked on GitHub so it should be fine
